pajbot/managers/connection.py

```diff
@@ -91,24 +91,22 @@ def start(self):
 
             return True
         except:
-            log.exception("babyrage")
+            log.exception("Connection failed, retrying")
```

Comment:
```suggestion
            log.exception("Failed to open connection, retrying")
```

"Connection failed" to me sounds like we had a connection, but it failed later